php/media/class-woocommercegallery.php

```diff
@@ -42,7 +42,7 @@ public function enqueue_gallery_library() {
 
 		if ( $assets ) {
 			$json_assets = wp_json_encode( $assets );
-			wp_add_inline_script( Gallery::GALLERY_LIBRARY_HANDLE, "cloudinaryGalleryConfig.mediaAssets = JSON.parse( '{$json_assets}' );" );
+			wp_add_inline_script( Gallery::GALLERY_LIBRARY_HANDLE, "CLD_GALLERY_CONFIG.mediaAssets = JSON.parse( '{$json_assets}' );" );
```

Comment:
Same question here -- is there a specific reason for using `JSON.parse()`?

Reply:
@dugajean You marked this as resolved but left in `JSON.parse()` -- is that intentional? Can you please confirm the usage of `JSON.parse()` here?

Reply:
@kasparsd Thanks for checking this. I was under the impression that I caught all of them 😄